setup.py

```diff
@@ -1,7 +1,64 @@
-"""Setup script for quconot project
+# Copyright 2022 Adam Glos
 
-This is file needed to allow installation in editable mode.
-"""
-from setuptools import setup
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 
-setup()
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import setup, find_packages
+
+VERSION = '0.01'

```

Comment:
this is strange. Perhaps it was supposed to be just 0.1?

Reply:
it is just  initial placeholder version. It will be replaced by a variable that auto-updates with releases